flight/Modules/PipXtreme/pipxtrememod.c

```diff
@@ -1,238 +0,0 @@
-/**
```

Comment:
pipxtrememod was a duplicate of systemmod with a few changes-- turning off configuration monitoring and changing the instance number of the rfm22b stats to update.

It was a question of either performing the same changes twice in the two different systemmods or factoring them into common code.

There is a small side effect: this makes the heartbeat LED on pipxtreme blink faster when the board it is talking to is armed.

(I still need to test and verify all of this).